routing/blindedpath/blinded_path.go

```diff
@@ -82,7 +82,16 @@ type BuildBlindedPathCfg struct {
 	MinFinalCLTVExpiryDelta uint32
 
 	// BlocksUntilExpiry is the number of blocks that this blinded path
-	// should remain valid for.
+	// should remain valid for. This is a relative number of blocks. This
+	// number in addition with a potential minimum cltv delta for the last
+	// hop and some block padding will be the payment constraint which is
+	// part of the blinded hop info. Every htlc using the provided blinded
+	// hops cannot have a higher cltv delta otherwise it will get rejected
+	// by the forwarding nodes or the final node.
+	//
+	// This number should at least be greater than the invoice expiry time
```

Comment:
Q: can this value be equal to the invoice's final cltv delta?

Reply:
Yes this value is not related to the final_cltv value, the final_cltv_value is added on top of this expiry when calculating the payment constraint for each blinded hop.